source/VideoPlayer.js

```diff
@@ -1873,7 +1877,7 @@
 		*/
 		timeUpdate: function(sender, e) {
 			//* Update _this.duration_ and _this.currentTime_
-			if (!e && e.srcElement) {
+			if (!e && e.srcElement || e.currentTime === undefined) {
```

Comment:
I found that event is sometimes comes with 'undefined' currentTime. I think null should be including typechecking because we can have time as 0.